netket/experimental/operator/_fermion_operator_2nd_base.py

```diff
@@ -47,9 +45,18 @@
 )
 
 
-class FermionOperator2nd(DiscreteOperator):
+# TODO is this a reasonable cutoff?
+@numba.jit(nopython=True)
+def _isclose(a, b, cutoff=1e-6):  # pragma: no cover
+    return np.abs(a - b) < cutoff
```

Comment:
```suggestion
```

Reply:
Seems like an artifact of the rebase. I would remove numba from the base class altogether, and use `np.isclose` instead of this.